lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3745,6 +3745,7 @@ JITManager::HandleServerCallResult(HRESULT hr, RemoteCallType callType)
         return true;
     case E_ABORT:
         throw Js::OperationAbortedException();
+    case 0x800705AF: // HRESULT for ERROR_COMMITMENT_LIMIT (The paging file is too small for this operation to complete)
```

Comment:
>0x800705AF [](start = 9, length = 10)

better to use
case HRESULT_FROM_WIN32(ERROR_COMMITMENT_LIMIT):
instead

Reply:
Good point. Updated.